src/stream/position.rs

```diff
@@ -233,6 +233,24 @@ impl Positioner<char> for SourcePosition {
     }
 }
 
+impl Positioner<u8> for SourcePosition {
```

Comment:
👍  This impl is still needed though so just revert the other changes

Reply:
removed all the other changes :+1: